js/_1btcxe.js

```diff
@@ -53,43 +53,40 @@ module.exports = class _1btcxe extends Exchange {
                     ],
                 },
             },
+            'markets': {
+                'BTC/AED': {'base': 'BTC', 'id': 'AED', 'quote': 'AED', 'symbol': 'BTC/AED'},
```

Comment:
this will break getbtc...

Reply:
Ah I see, didn't release it subclass _1btcxe >:

Reply:
Yep, having them in fetchMarkets is a special case when both the base and the child have them hardcoded...

Reply:
Haven't come up with a better way to handle it yet.